.github/workflows/links-fail-fast.yml

```
@@ -0,0 +1,17 @@
+name: Links (Fail Fast)
```

What does "(Fail Fast)" mean in this context? Why isn't it just links?

---

Renamed to links